tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/move_instruction.h

```diff
@@ -54,16 +55,17 @@ class MoveInstruction
   using Ptr = std::shared_ptr<MoveInstruction>;
   using ConstPtr = std::shared_ptr<const MoveInstruction>;
 
-  MoveInstruction(Waypoint waypoint, MoveInstructionType type, const std::string& profile = "DEFAULT");
+  MoveInstruction(Waypoint waypoint,
+                  MoveInstructionType type,
+                  const std::string& profile = "DEFAULT",
+                  ManipulatorInfo manipulator_info = ManipulatorInfo());
```

Comment:
Should this be a required by the user to provide? The only field that must be field out is `manipulator`

Reply:
Maybe. I guess it's unlikely that things would work if you don't fill it out, but you could set it later. 

Would you rather?
```
  MoveInstruction(Waypoint waypoint,
                  MoveInstructionType type,
                 ManipulatorInfo manipulator_info
                  const std::string& profile = "DEFAULT");
```